### PR TITLE
Create tests for assign/remove command

### PR DESCRIPTION
**Pull request by Joanna-YJA**
## Description
This pull request implements JavaFX observable list interface with the User Interface and creates tests for assign and remove commands.

## Checks
- [] No errors when running the tests
- [] Build and checkstyle passes
- [] Run the app and ran 2 commands without failing
- [] Written the baseline test cases for the PR

## Changelog
- []
- []
- []

### CODE REVIEW

File: src/main/java/seedu/address/ui/SuggestionTemplates.java [hunk 1]
```diff
@@ -433,7 +433,7 @@ public static final String getString(TextFlow template) {
                 .stream()
                 .map(textElement -> (Text) textElement)
                 .map(t -> t.getText())
-                .filter(s -> !s.matches("\\[[A-Za-z{}_ ]+\\] |[A-Z_{} ]+|\\([A-Za-z_ ]+\\)"))
+                .filter(s -> !s.matches("\\[[A-Za-z{}_ ]+\\] |[A-Z_{} ]+|\\([A-Za-z_. ]+\\)"))
```
Comment (hcwong) [inline src/main/java/seedu/address/ui/SuggestionTemplates.java]: Seems like this is repeated from the previous PR

File: src/test/test.iml [hunk 2]
```diff
@@ -8,5 +8,21 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="module" module-name="main" />
+    <orderEntry type="module-library" scope="TEST">
```
Comment (hcwong) [inline src/test/test.iml]: Whats this?

File: src/main/java/seedu/address/ui/SuggestionTemplates.java (hunk 1 above)
Comment (Joanna-YJA) [reply src/main/java/seedu/address/ui/SuggestionTemplates.java]: yeap becuase I have yet to merge the previous pr in:)

File: src/test/test.iml [hunk 3]
```diff
@@ -8,5 +8,19 @@
...
+          <root url="jar://$MAVEN_REPOSITORY$/org/apiguardian/apiguardian-api/1.0.0/apiguardian-api-1.0.0.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/opentest4j/opentest4j/1.1.1/opentest4j-1.1.1.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/platform/junit-platform-commons/1.4.2/junit-platform-commons-1.4.2.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter-params/5.4.2/junit-jupiter-params-5.4.2.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/jupiter/junit-jupiter-engine/5.4.2/junit-jupiter-engine-5.4.2.jar!/" />
+          <root url="jar://$MAVEN_REPOSITORY$/org/junit/platform/junit-platform-engine/1.4.2/junit-platform-engine-1.4.2.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
```
Comment (brianyenna) [inline src/test/test.iml]: ALso, can i check what is this for

**Pull request merged**